spec/high_voltage/configuration_spec.rb

```diff
@@ -35,7 +35,7 @@
     it { expect(HighVoltage.routes).to eq config_value }
   end
 
-  describe '#action_caching=' do
+  describe '#action_caching=true' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.